app-backup/spideroak-bin/spideroak-bin-7.5.0-r3.ebuild

```diff
@@ -0,0 +1,105 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop prefix unpacker
+
+DESCRIPTION="Secure free online backup, storage, and sharing system"
+HOMEPAGE="https://spideroak.com"
+
+SRC_URI_BASE="https://spideroak.com/release/spideroak"
+SRC_URI="
+	x86? ( ${SRC_URI_BASE}/deb_x86 -> ${P}_x86.deb )
+	amd64? ( ${SRC_URI_BASE}/deb_x64 -> ${P}_amd64.deb )
+"
+S=${WORKDIR}
```

Comment:
I suggest: `S="${WORKDIR}"`

Reply:
Ups, you are totally right, I missed it :P